lib/spack/spack/test/cmd/blame.py

```diff
@@ -70,6 +70,7 @@ def test_blame_json(mock_packages):
 
 def test_blame_by_git(mock_packages, capfd):
     """Sanity check the blame command to make sure it works."""
+    # TODO, avoid noisy output in CI: https://github.com/spack/spack/pull/27659
```

Comment:
Sorry but I'm having trouble connecting this explanation to https://github.com/spack/spack/pull/27659#issuecomment-1062293991:

* I think the problem is more like the output isn't noisy enough - it doesn't get captured and returned by the `Executable` invocation
* You seem to be referring to another issue here (which I wasn't even aware of): the CI output has increased - do you mean for example that `spack unit-test ...` prints output from these commands (interleaved with the normal test reporting which just looks like a bunch of `.`'s)?

Reply:
Yes, this was already the case for the log test. But again, it's better to be verbose than get nothing during spurious failures.

Reply:
I agree, although to be clear what I mean is that this comment does not address my original concern (which was: "why do we have to set `capfd.disabled` in these specific circumstances") - could you update the comment to mention that as well?